backend/init_data/skills/document/provider.py

```
@@ -0,0 +1,127 @@
+            ValueError: If tool_name is unknown
+        """
+        import logging
+
+        logger = logging.getLogger(__name__)
+
+        logger.info(
+            f"[DocumentProvider] Creating tool: {tool_name}, "
+            f"task_id={context.task_id}, user_id={context.user_id}, user_name={context.user_name}"
+        )
+
+        config = tool_config or {}
```

Ruff (0.14.13)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)